src/mapView.ts

```diff
@@ -404,20 +458,35 @@ export class MapView extends ItemView {
 			await editorAction(editor);
 	}
 
+	/**
+	 * Open the marker in an editor instance
```

Comment:
I think you need to standardise your terminology because you are using location here to mean a position in a file and you are also using location to refer to a geographical location. I suggest renaming the latter terminology to either geolocation or just coordinate.